### PR TITLE
Create de_de.lang

### PR DESCRIPTION
**Pull request by realmayus**
Created German language file (.lang)

### CODE REVIEW

File: src/main/resources/assets/bashfiremakerock/lang/de_de.lang [hunk 1]
```diff
@@ -0,0 +1,9 @@
+config.bashfiremakerock:title=Hauptmenü
+config.bashfiremakerock:chance=Chance auf Feuer
+config.bashfiremakerock:blocks=Blöcke
+config.bashfiremakerock:blocks.allow=Erlaubte Liste
+config.bashfiremakerock:start=Minimum an Klicks für Feuer
+config.bashfiremakerock:delay.min=Verzögerung zwischen Klicks
+config.bashfiremakerock:delay.max=Verzögerung um Klicks zu merken
+
+bashfiremakerock:error.event.click=Fehlerhandlungsereignis!! Benachrichtige einen Administrator, den Konosolenlog einzusehen.
```
Comment (bl4ckscor3) [inline src/main/resources/assets/bashfiremakerock/lang/de_de.lang]: The `Handling` in `Error Handling Click Event` is a verb. Should be something like `Fehler beim Verarbeiten des Klicks`

File: src/main/resources/assets/bashfiremakerock/lang/de_de.lang [hunk 2]
```diff
@@ -0,0 +1,9 @@
+config.bashfiremakerock:title=Hauptmenü
```
Comment (bl4ckscor3) [inline src/main/resources/assets/bashfiremakerock/lang/de_de.lang]: It's not really a main menu, it's the main category of the config file.

**Review by bl4ckscor3 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/bashfiremakerock/lang/de_de.lang (hunk 1 above)
Comment (realmayus) [reply src/main/resources/assets/bashfiremakerock/lang/de_de.lang]: Done

File: src/main/resources/assets/bashfiremakerock/lang/de_de.lang (hunk 2 above)
Comment (realmayus) [reply src/main/resources/assets/bashfiremakerock/lang/de_de.lang]: Changed it to "Allgemein"

**Pull request merged**